nest-city-account/src/physical-entity/physical-entity.service.ts

```diff
@@ -96,21 +95,26 @@ export class PhysicalEntityService {
     return entity
   }
 
-  async updateUriAndEdeskFromUpvs(upvsInput: UpvsIdentityByUriServiceCreateManyParam) {
+  async checkUriAndUpdateEdeskFromUpvs(upvsInput: {
+    physicalEntityId?: string | null | undefined
```

Comment:
```suggestion
    physicalEntityId?: string | null
```

this should be the same I suppose :D just so we do not have 3 different types